src/radio/radio-group.component.ts

```diff
@@ -0,0 +1,407 @@
+	 * @type {EventEmitter<RadioChange>}
+	 * @memberof RadioGroup
+	 */
+	@Output() change: EventEmitter<RadioChange> = new EventEmitter<RadioChange>();
+
+	/**
+	 * The `RadioComponent` input items in the `RadioGroup`.
+	 * @type {QueryList<RadioComponent>}
+	 * @memberof RadioGroup
+	 */
+	// tslint:disable-next-line:no-forward-ref
+	@ContentChildren(forwardRef(() => RadioComponent)) radios: QueryList<RadioComponent>;
+
+	/**
+	 * Determines the render size of the `RadioComponent` inputs within the group.
+	 * (size `"default"` is being deprecated as of next major release, please use `"md"` instead)
+	 * @type {("sm" | "md" | "default")}
+	 * @memberof RadioGroup
+	 */
+	@Input() size: "sm" | "md" | "default" = "md";
+
+	/**
+	 * Returns the `RadioComponent` that is selected within the `RadioGroup`.
+	 * @readonly
+	 * @memberof RadioGroup
+	 */
+	@Input()
+	get selected() {
+		return this._selected;
+	}
+	/**
+	 * Sets the passed in `RadioComponent` item as the selected input within the `RadioGroup`.
+	 * @param selected
+	 * @memberof RadioGroup
+	 */
+	set selected(selected: RadioComponent | null) {
+		this._selected = selected;
+		this.value = selected ? selected.value : null;
+		this.checkSelectedRadio();
+	}
+
+	/**
+	 * Returns the value/state of the selected `RadioComponent` within the `RadioGroup`.
+	 * @readonly
+	 * @memberof RadioGroup
+	 */
+	@Input()
+	get value() {
+		return this._value;
+	}
+	/**
+	 * Sets the value/state of the selected `RadioComponent` within the `RadioGroup` to the passed in value.
+	 * @param {newValue}
+	 * @memberof RadioGroup
+	 */
+	set value(newValue: any) {
+		if (this._value !== newValue) {
+			this._value = newValue;
+
+			this.updateSelectedRadioFromValue();
+			this.checkSelectedRadio();
+		}
+	}
+
+	/**
+	 * Returns the associated name of the `RadioGroup`.
+	 * @readonly
+	 * @memberof RadioGroup
+	 */
+	@Input()
+	get name() {
+		return this._name;
+	}
+	/**
+	 * Replaces the name associated with the `RadioGroup` with the provided parameter.
+	 * @param name
+	 * @memberof RadioGroup
+	 */
+	set name(name: string) {
+		this._name = name;
+		this.updateRadioNames();
+	}
+
+	/**
+	 * Returns the disabled value in the `RadioGroup` if there is one.
+	 * @readonly
+	 * @memberof RadioGroup
+	 */
+	@Input()
+	get disabled() {
+		return this._disabled;
+	}
+	/**
+	 * Updates the disabled value using the provided parameter and marks the radios to be checked for
+	 * changes.
+	 * @param value
+	 * @memberof RadioGroup
+	 */
+	set disabled(value) {
+		this._disabled = value;
+		this.markRadiosForCheck();
+	}
+
+	/**
+	 * Binds 'radiogroup' value to the role attribute for `RadioGroup`.
+	 * @memberof RadioGroup
+	 */
+	@HostBinding("attr.role") role = "radiogroup";
+
+	/**
+	 * Binds 'bx--radio-button-group' value to the class for `RadioGroup`.
+	 * @memberof RadioGroup
+	 */
+	@HostBinding("class") class = "bx--radio-button-group";
```

Comment:
Let's allow users to add their own classes. ;)
```suggestion
	@HostBinding("class.bx--radio-button-group") radioButtonGroupClass = true;
```